src/config/example.toml

```diff
@@ -87,12 +87,39 @@ object_store_streaming_upload_timeout_ms = 600000
 object_store_upload_timeout_ms = 3600000
 object_store_read_timeout_ms = 3600000
 
-[storage.file_cache]
+[storage.data_file_cache]
```

Comment:
Ask just in case: Is there any influences of the breaking change? (IIUC it's a undocumented experimental feature)

Reply:
Thanks for the reminder. The new engine is not compatible with the old one. And as it is still an experimental feature, the format of the new engine can still include broken changes in the future until the feature is stable. For the old users, they can safely remove the data of the old engine and use the new one. The only consideration is just the cache warm-up time.